api.js

```diff
@@ -27,4 +27,14 @@ export const getComments = (article_id) => {
         return data
       })
 
+}
+
+export const voteComment = (article_id) => {
+  
+  return ncnewsApi
+    .patch(`/articles/${article_id}`, { inc_votes: 1 })
+    .then((response) => {
+      console.log(response);
+      
+    })
```

Comment:
If this response is not needed I would recommend removing the .then(...) from here as well